OpenRA.Mods.Common/FileSystem/ContentInstallerFileSystemLoader.cs

```diff
@@ -13,19 +13,24 @@
 
 namespace OpenRA.Mods.Common.FileSystem
 {
+	[Desc("A file system that deals with loading assets installed from external sources.")]
```

Comment:
I'd suggest

> "A file system that can load game assets installed by the user into their support directory"

Reply:
"ContentInstaller" kind of suggests it's the one doing the installing, when in fact it depends on the content already being installed by someone else and this just loads it. Perhaps a rename of the class is in order?

Reply:
I intended it as "ContentInstallerFileSystem"-Loader (i.e. load a file system populated by the content installer as opposed to "ContentInstaller"-"FileSystemLoader", but I can indeed see the potential for confusion.

Reply:
I didn't even register the "Loader" part.
To the users it is `FileSystem: ContentInstallerFileSystem`, so a "ContentInstaller FileSystem", sounding like it is a file system that installs content 🤷‍♂️